src/test/java/org/umssdiplo/automationv01/stepdefinitionproject/HomeSteps/HomeSteps.java

```diff
@@ -0,0 +1,26 @@
+package org.umssdiplo.automationv01.stepdefinitionproject.HomeSteps;
+
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import org.umssdiplo.automationv01.core.managepage.Home.SSIAHome;
+
+/**
+ * @Author: Lizeth Salazar
+ */
+public class HomeSteps {
```

Comment:
you need a constructor in your page objects because when you are going to load this page, you need to wait for an WebElement especific.
CommonEvents.isVisible(yourWebElment). or you can implement a new method that is going to wait for an especific WebElement.